lib/playground.dart

```diff
@@ -533,11 +538,28 @@ class Playground implements GistContainer, GistController {
       overlay.visible = false;
     });
   }
-
+  
+  
+  /// Perform static analysis of the source code. Return a summary future.
+  Future<String> _formSummary() {
+      SourceRequest input = new SourceRequest()..source = _context.dartSource;
+      Future request = dartServices.analyze(input).timeout(serviceCallTimeout);
+      _analysisRequest = request;
```

Comment:
We can remove this line; it was relevant in the `_performAnalysis` method, but not here.